tests/OmniSharp.MSBuild.Tests/AbstractMSBuildTestFixture.cs

```diff
@@ -47,7 +45,7 @@ protected OmniSharpTestHost CreateMSBuildTestHost(string path, IEnumerable<Expor
             IConfiguration configurationData = null)
         {
             var environment = new OmniSharpEnvironment(path, logLevel: LogLevel.Trace);
-            var serviceProvider = TestServiceProvider.Create(this.TestOutput, environment, this.LoggerFactory, _assemblyLoader, _analyzerAssemblyLoader, _msbuildLocator,
+            var serviceProvider = TestServiceProvider.Create(this.TestOutput, environment, this.LoggerFactory, _assemblyLoader, ShadowCopyAnalyzerAssemblyLoader.CreateShadowCopyLoader(), _msbuildLocator,
```

Comment:
We can no longer use one assembly loader for all the test runs because it now gets disposed with the other test services. See https://github.com/dotnet/roslyn/pull/74845